app/controllers/multifactor_auths_controller.rb

```diff
@@ -13,8 +13,8 @@ class MultifactorAuthsController < ApplicationController
 
   def new
     @seed = ROTP::Base32.random_base32
-    session[:mfa_seed] = @seed
-    session[:mfa_seed_expire] = Gemcutter::MFA_KEY_EXPIRY.from_now.utc.to_i
+    session[:totp_seed] = @seed
+    session[:totp_seed_expire] = Gemcutter::MFA_KEY_EXPIRY.from_now.utc.to_i
```

Comment:
I'm guessing that there's a chance that a couple of errors will be thrown if someone is in the process of setting up new MFA. That person would just be very unlucky 😅